scripts/src/blockchain/channelsGenerator.ts

```diff
@@ -0,0 +1,93 @@
+import {KeyPair} from "./keyPair";
+import * as crypto from "crypto";
+import {TxSender} from "./txSender";
+import {Network, Operation} from "@kinecosystem/kin-base";
+import {Environment} from "../environment";
+import {Server} from "@kinecosystem/kin-sdk";
+import {AccountDataRetriever, IAccountDataRetriever} from "./accountDataRetriever";
+import {BlockchainInfoRetriever, IBlockchainInfoRetriever} from "./blockchainInfoRetriever";
+
+export namespace Channels {
```

Comment:
Add the channels to the index.d.ts

Reply:
done with various other overlooked classes